src/doc/en/installation/meson.rst

```diff
@@ -65,11 +65,23 @@ or run the tests with ``./sage -t``.
     This means that any Sage-the-distribution commands such as ``sage -i`` 
     will not work.
 
+.. NOTE::
+
+    By default, ``ninja`` will automatically determine the number of jobs to
+    run in parallel based on the number of CPU available. This can be adjusted
+    by passing ``--config-settings=compile-args=-jN`` to ``pip install``,
+    which will pass ``-jN`` to ``ninja``.
```

Comment:
```suggestion
    By default, Meson will automatically determine the number of jobs to
    run in parallel based on the number of CPU available. This can be adjusted
    by passing ``--config-settings=compile-args=-jN`` to ``pip install``.
```

That ninja is used behind the scenes is, at this general level, an "implementation detail" I would say.

Reply:
But the problem is if I understood correctly `compile-args` is passed to ninja directly (instead of to meson), so the user need to know that to know which flags are available. Need double check.

Reply:
I'm fine with adding more details (and eg mentioning the nice trick using `--help`) further below in the background section. This initial section, however, is designed for average developers that don't want to deep-dive into meson (and thus may not even know that meson uses ninja under the hood).